types/ioredis-mock/index.d.ts

```diff
@@ -8,15 +8,27 @@ import ioredis = require('ioredis');
 
 export type RedisOptions = { data?: Record<string, unknown> } & ioredis.RedisOptions;
 
+export type RedisClusterOptions = {
+    redisOptions: Omit<
```

Comment:
For reference this was taken from: https://github.com/luin/ioredis/blob/v5.3.2/lib/cluster/ClusterOptions.ts#L127